### PR TITLE
Wiki Reorg: Cards Learn Section

### PR DESCRIPTION
**Pull request by filippoweb3**
See [this issue](https://github.com/w3f/polkadot-wiki/issues/4376) and [this google doc](https://docs.google.com/document/d/1Fl8ReBIzcaQEOyF6s03d7MEcgk8aTVEt7xjmzAk_k5I/edit) for more information about the Wiki reorg.

This PR addresses the addition of cards for the whole Learn Section of the Wiki. All .md files have been edited only in `description` and sometimes `sidebar-label`. The .js sidebar file has been edited to accommodate the cards for all subsections. The `learn-index` has been deleted and everything redirected to `/learn` (i.e. the page displaying all cards for the learn section).

### CODE REVIEW

**Comment by filippoweb3** [comment]
@CrackTheCode016 I know you worked on the `learn-index` page. Let me know if you agree to delete it. I just thought that the new `learn` page is easier to navigate and still contains a short description of each subsection. I tried to modify the descriptions to make them shorter without changing their meaning. I tried to navigate within the wiki, and to me, it seems quicker and more intuitive. In the nested menu in the top-left, every element has now a link. So, it is easier to just click on it and go back to previous subsections with cards. Also, the link to subsections with cards is always the name of the subsection like for example `/learn` for Learn. We can still discuss if we prefer to have `learn-index` and/or for subsections have `/accounts` or `learn-accounts` -> this will require changing some links as there is for example already a page called `learn-accounts`.

**Comment by CrackTheCode016** [comment]
@filippoweb3 This is so much cleaner, I love it.

We can remove `learn-index` and use this layout, IMO it is also way more intuitive to navigate

**Review by CrackTheCode016 — APPROVED** [review]
LGTM, really nice work looks great (tested locally too)!  One super tiny nit is that every letter is capitalized - not really a dealbreaker but just making note of it ;)

**Comment by filippoweb3** [comment]
@DrW3RK this has been approved by Bader but I wait anyway for your green light ;)

File: docs/general/grants.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 id: grants
 title: Grants
 sidebar_label: Grants
-description: Learn about Grant Programs in the Polkadot ecosystem
+description: The Grant Program in the Polkadot Ecosystem.
```
Comment (profradha) [inline docs/general/grants.md]:
```suggestion
description: The Grant Programs in the Polkadot Ecosystem.
```

File: docs/learn/learn-balance-transfers.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ id: learn-balance-transfers
 title: Balance Transfers
 sidebar_label: Balances Transfers
 keywords: [balance, transfers, transaction, signing]
-description: Steps on how to perform balance transfers.
+description: Perform Balances Transfers between Accounts.
```
Comment (profradha) [inline docs/learn/learn-balance-transfers.md]:
```suggestion
description: Perform Balance Transfers between Accounts.
```

File: docs/learn/learn-consensus.md [hunk 3]
```diff
@@ -2,7 +2,7 @@
 id: learn-consensus
 title: Polkadot Consensus
 sidebar_label: Consensus
-description: An explanation of the consensus model used in Polkadot.
+description: The Consensus Mechanism used in Polkadot.
```
Comment (profradha) [inline docs/learn/learn-consensus.md]:
```suggestion
description: The Consensus Mechanism of Polkadot.
```

File: docs/learn/learn-crowdloans.md [hunk 4]
```diff
@@ -2,7 +2,7 @@
 id: learn-crowdloans
 title: Parachain Crowdloans
 sidebar_label: Parachain Crowdloans
-description: Learn about crowdloans and how to interact with one.
+description: Polkadot's Crowdloands and How to Participate.
```
Comment (profradha) [inline docs/learn/learn-crowdloans.md]:
```suggestion
description: Polkadot's Crowdloans and How to Participate.
```

File: docs/learn/learn-cryptography.md [hunk 5]
```diff
@@ -2,7 +2,7 @@
 id: learn-cryptography
 title: Cryptography on Polkadot
 sidebar_label: Cryptography
-description: An overview of the cryptographic functions used in Polkadot.
+description: Cryptographic Functions used in Polkadot's Design.
```
Comment (profradha) [inline docs/learn/learn-cryptography.md]: Not just design. In code as well.

```suggestion
description: Cryptographic Functions used in Polkadot.
```

File: docs/learn/learn-nft.md [hunk 6]
```diff
@@ -2,7 +2,7 @@
 id: learn-nft
 title: NFTs
 sidebar_label: Introduction to NFTs
-description: An explanation of the NFT landscape in the Polkadot ecosystem.
+description: The NFT Landscape in the Polkadot Ecosystem.
```
Comment (profradha) [inline docs/learn/learn-nft.md]:
```suggestion
description: The NFT Landscape of the Polkadot Ecosystem.
```

File: docs/learn/learn-nominator.md [hunk 7]
```diff
@@ -2,7 +2,7 @@
 id: learn-nominator
 title: Nominator
 sidebar_label: Nominator
-description: Learn about what it means to be a nominator.
+description: Role of Nominators within the Polkadot Ecosystem.
```
Comment (profradha) [inline docs/learn/learn-nominator.md]:
```suggestion
description: Role of Nominators on Polkadot.
```

File: docs/learn/learn-parachains-protocol.md [hunk 8]
```diff
@@ -2,7 +2,7 @@
 id: learn-parachains-protocol
 title: Parachains' Protocol Overview
 sidebar_label: Protocol Overview
-description: High level view of actors and protocols involved in running parachains on Polkadot.
+description: Actors and Protocols shaping Polkadot's Block Road to Finality.
```
Comment (profradha) [inline docs/learn/learn-parachains-protocol.md]:
```suggestion
description: Actors and Protocols involved in Polkadot and its Parachains' Block Finality.
```

File: docs/learn/learn-phragmen.md [hunk 9]
```diff
@@ -2,7 +2,7 @@
 id: learn-phragmen
 title: NPoS Election Algorithms
 sidebar_label: NPoS Election Algorithms
-description: Learn about the election method used on Polkadot's Nominated Proof of Staking.
+description: The Election Method used in Polkadot's NPoS Mechanism.
```
Comment (profradha) [inline docs/learn/learn-phragmen.md]:
```suggestion
description: The Election Methods used in Polkadot's NPoS Mechanism.
```

File: docs/learn/learn-polkadot-opengov.md [hunk 10]
```diff
@@ -2,7 +2,7 @@
 id: learn-polkadot-opengov
 title: Polkadot OpenGov
 sidebar_label: Polkadot OpenGov
-description: Learn about Polkadot’s next generation of decentralised governance.
+description: Polkadot’s Next-gen of Decentralized Governance.
```
Comment (profradha) [inline docs/learn/learn-polkadot-opengov.md]:
```suggestion
description: Polkadot’s Latest Model for Decentralized Governance.
```

File: docs/learn/learn-teleport.md [hunk 11]
```diff
@@ -2,7 +2,7 @@
 id: learn-teleport
 title: Teleporting Assets
 sidebar_label: Teleporting Assets
-description: Steps on how to teleport assets across parachains.
+description: Teleport Assets between Parachains and Relay Chains.
```
Comment (profradha) [inline docs/learn/learn-teleport.md]:
```suggestion
description: Teleport Assets between Parachains and Relay Chain.
```

**Review by DrW3RK — APPROVED** [review]
Some minor suggestions. Feel free to accept/improve on them.

**Pull request merged**